6. Developer Guides/REST API/Users/list.md

```diff
@@ -9,6 +9,15 @@ Gets all of the users in the system and their information, the result is only li
 | :--- | :--- | :--- |
 | `/api/v1/users.list` | `yes` | `GET` |
 
+## Query Parameters
```

Comment:
Can you remove the offset, count, and sort ones as they're documented here: https://github.com/RocketChat/Rocket.Chat.Docs/blob/master/6.%20Developer%20Guides/REST%20API/Offset%20and%20Count%20Info.md then put a link to it somewhere in this document? Also, the fields and query ones need a dedicated document to further explain what features are enabled on them.. As the example provided will actually fail due to not being valid json format.